viral/sequence_utils.py

```
@@ -0,0 +1,859 @@
+
+
+def create_dummy_ppm_more_complex(n_spatial_bins: int, n_time_bins: int) -> np.ndarray:
+    # (n_time_bins, n_spatial_bins)
+    ppm = np.zeros(shape=(n_time_bins, n_spatial_bins))
+    np.fill_diagonal(ppm, 0.5)
+    ppm[3, 5] = 0.7
+    return ppm
+
+
+def compare_radon_against_matlab() -> None:
+    from scipy.io import savemat, loadmat
```

Ruff (0.14.10)

Redefinition of unused `loadmat` from line 6

Remove definition: `loadmat`

(F811)